doc/install/single-server-deployment.rst

```diff
@@ -0,0 +1,85 @@
+
+
+====================================
+Bootstrap a Cluster on a Single Host
```

Comment:
Hi @khodaparast, there's a section in the docs for different deployment scenarios (one subsection for single host, but it's quite short at the moment): https://docs.ceph.com/en/latest/cephadm/install/#different-deployment-scenarios
I think it would be great to have your docs in that section

Reply:
Hi @melissa-kun-li, 

Thank you for the suggestion. The documentation is now in the recommended location. 
Please let me know if it requires any changes.